docs/portal/registry-users/viewing-users.md

```diff
@@ -9,8 +9,8 @@ The following procedure describes how to view information about registry users.
 
 | **Element Number**       | **Description**                               |
 | -------------------------|-----------------------------------------------| 
-1                                                                                                       | The Basic Information area shows basic information about the registry user. To [view a project](</docs/portal/image-registry/viewing-projects.md>) associated with the registry user, click a project in the table within the **Access to Projects** field.                                            |
-| 2                        | Click this button to return to the Registry Users page.                                                                 |
-| 3                        | Click this button to change the registry user's [password](</docs/portal/registry-users/changing-passwords.md>).            |
-| 4                        | Click this button to [delete](</docs/portal/registry-users/deleting-users.md>) the registry user. |
-| 5                        | Click this button to [update](</docs/portal/registry-users/updating-users.md>) the registry user.                       |
+<p style="text-align: center;">1<p>                                                                                                       | The Basic Information area shows basic information about the registry user. To [view a project](</docs/portal/image-registry/viewing-projects.md>) associated with the registry user, click a project in the table within the **Access to Projects** field.                                            |
```

Comment:
The preview shows there are errors. Please check
![image](https://user-images.githubusercontent.com/43203744/105330282-144f3e80-5c0d-11eb-8725-d6f2d848e2de.png)